src/common/utils/process_path.h

```diff
@@ -53,12 +58,25 @@ inline std::wstring get_process_path(HWND window) noexcept
                 }
             },
             reinterpret_cast<LPARAM>(&new_pid));
+
         // If we have a new pid, get the new name.
         if (new_pid != pid)
         {
+            retryLoopCounter = 0;
             return get_process_path(new_pid);
         }
+        else
+        {
+            if (retryLoopCounter < 10)
+            {
+                retryLoopCounter++;
+                std::this_thread::sleep_for(std::chrono::milliseconds(100));
+                return get_process_path(pid);
+            }
```

Comment:
Can you please explain how this retry works? Shouldn't it be a loop with the `EnumChildWindows` function inside?

Reply:
I made it recursive but after your question realized that with a loop it would be more clear. Thank you!